tests/cnf/core/network/metallb/tests/common.go

```diff
@@ -466,11 +486,15 @@ func validatePrefix(
 
 	bgpStatus, err := frr.GetBGPStatus(masterNodeFRRPod, strings.ToLower(ipProtoVersion), "test")
 	Expect(err).ToNot(HaveOccurred(), "Failed to verify bgp status")
-	_, subnet, err := net.ParseCIDR(fmt.Sprintf("%s/%d", addressPool[0], prefix))
+
+	_, subnet, err = net.ParseCIDR(fmt.Sprintf("%s/%d", addressPool[0], prefix))
```

Comment:
why do need this chnage?

Reply:
I dont. Removed. I had add a func that used subnet but didnt revert it back.